### ISSUE TITLE
Normalizing THE

### ISSUE THREAD

**Issue by bkrawiec** [CopticScriptorium/normalizer#4]
@amir-zeldes @ctschroeder Should the normalizer be expected to change ⲛⲑⲉ to  ⲛ-ⲧ-ϩⲉ? I just ran the normalizer on a section of YB and it did not make this change.

**Comment by amir-zeldes** [comment]
No, the normalizer doesn’t alter thetas in this way (it generally does not revise segmentation, just works line by line). The tokenizer should be catching this.

**Comment by bkrawiec** [comment]
The tokenizer was not catching it when I converted YB (unpiped section). I will double check when I work on GL.
